smart_contract/executor/derive/src/default.rs

```diff
@@ -142,21 +142,20 @@ pub fn impl_derive_visit(emitter: &mut Emitter, input: &syn::DeriveInput) -> Tok
         "fn visit_transfer_asset_definition(operation: &Transfer<Account, AssetDefinitionId, Account>)",
         "fn visit_set_asset_definition_key_value(operation: &SetKeyValue<AssetDefinition>)",
         "fn visit_remove_asset_definition_key_value(operation: &RemoveKeyValue<AssetDefinition>)",
-        "fn visit_grant_account_permission(operation: &Grant<Permission, Account>)",
-        "fn visit_revoke_account_permission(operation: &Revoke<Permission, Account>)",
+        "fn visit_grant_account_permission(operation: &Grant<PermissionObject, Account>)",
```

Comment:
```suggestion
        "fn visit_grant_account_permission(operation: &Grant<Permission, Account>)",
```

I would prefer if we kept the naming. I know it collides with the trait name, but the 2 will never intersect in user code